package.json

```diff
@@ -127,5 +127,8 @@
   "dependencies": {
     "@react-docgen/cli": "^2.0.4",
     "debounce": "^2.1.1"
+  },
+  "optionalDependencies": {
```

Comment:
GitHub Actions fails to install this package. Even if it fails, the process can continue.

ref: https://github.com/npm/cli/issues/4828